### ISSUE TITLE
paper-ripple accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-ripple#16]
Keyboard only navigation: For the most part, this is working fine from an accessibility standpoint. The one thing to mention though is that I cannot get to the buttons using just the keyboard (without a screen reader on), so I can therefore not access and activate the buttons to see the ripple without using the mouse. 

Keyboard/Screen reader navigation: On Chrome OS / Chrome / ChromeVox, and Windows / Chrome / NVDA, I can navigate the buttons using the screen readers, but when I press enter/space to activate, I cannot trigger the ripple using just the keyboard. This only works using mouse for me.

**Comment by cdata** [comment]
The "buttons" and other widgets used in the ripple demo are styled to look like special widgets, but are really just divs. They don't have a role, they aren't stateful and they aren't aware of keyboard input.

We built the demo this way to test ripples in different containers (incidentally, the containers look like widgets). In cases where the ripple _does_ respond to keyboard input (e.g., paper-button) it is because paper-button is aware of keyboard input and can forward interactions to the ripple.

**Comment by lpalmaro** [comment]
Got it. Thanks for the clarification! This should be fine then.
